src/openassetio-python/tests/cmodule/gil/test_host_gil.py

```diff
@@ -0,0 +1,110 @@
+#
+#   Copyright 2023 The Foundry Visionmongers Ltd
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+"""
+Testing that Host/HostInterface methods release the GIL.
+"""
+# pylint: disable=redefined-outer-name
+# pylint: disable=invalid-name,c-extension-no-member
+# pylint: disable=missing-class-docstring,missing-function-docstring
+import pytest
+
+from openassetio import _openassetio_test
+from openassetio.hostApi import HostInterface
+from openassetio.managerApi import Host
+
+
+class Test_HostInterface:
+    """
+    See docstring for similar test under `gil/Test_ManagerInterface.py`
+    for details on how these tests are structured.
+    """
```

Comment:
[Consider][Minor] A bit annoying to read this if this file is where you land during browsing. Maybe duping the docs is a good thing here.

Reply:
Added a little snippet above in a70227cf

Reply:
Something is better than nothing. :+1: